ui/webui/test/helpers/installer.py

```diff
@@ -88,6 +88,16 @@ def back(self, should_fail=False, subpage=False):
         else:
             self.wait_current_page(self.steps[current_step_id-1])
 
+    @log_step()
+    def check_back_disabled(self, disabled=True):
```

Comment:
I don't see this beign re-usable as it's something we don't need to test again. I would not add it to helpers, just add this one line check in the test.